feature-libs/checkout/base/components/checkout-payment-method/checkout-payment-method.component.spec.ts

```diff
@@ -617,5 +617,82 @@ describe('CheckoutPaymentMethodComponent', () => {
       );
       expect(globalMessageService.add).not.toHaveBeenCalled();
     });
+
+    describe('createCard().role', () => {
+      let paymentMethod1: PaymentDetails;
+      beforeEach(() => {
+        paymentMethod1= {
+          id: 'selected payment method',
+          accountHolderName: 'Name',
+          cardNumber: '123456789',
+          cardType: {
+            code: 'Visa',
+            name: 'Visa',
+          },
+          expiryMonth: '01',
+          expiryYear: '2022',
+          cvn: '123',
+          defaultPayment: true,
+        };
+      });
+
+      it('should be set to "region" for selected cards when feature flag is enabled', () => {
+        spyOn(featureConfig, 'isEnabled').and.returnValue(true);
+        
+        expect(component['createCard'](
+          paymentMethod1,
+          {
+            textDefaultPaymentMethod: '✓ DEFAULT',
+            textExpires: 'Expires',
+            textUseThisPayment: 'Use this payment',
+            textSelected: 'Selected',
+          },
+          paymentMethod1
+        ).role).toEqual('region');
+      });
+
+      it('should be set to "region" when feature flag is disabled', () => {
```

Comment:
We don't really need to test for the feature flags because they will be removed anyway next major and we can always assume they are active for unit tests. The main thing to test here was that `isSelected` sets the role correctly. Is it possible to remove any reference to feature flags here?